frontend/packages/ux-editor/src/components/config/EditFormContainer.tsx

```diff
@@ -127,110 +106,100 @@ export const EditFormContainer = ({
     });
   };
 
-  return container.type === ComponentType.Group ? (
-    <LegacyFieldSet className={classes.fieldset}>
-      <FormField
-        id={container.id}
-        label={t('ux_editor.modal_properties_group_change_id')}
-        value={container.id}
-        propertyPath='definitions/component/properties/id'
-        customValidationRules={(value: string) => {
-          if (value !== container.id && idExists(value, components, containers)) {
-            return 'unique';
-          }
-        }}
-        customValidationMessages={(errorCode: string) => {
-          if (errorCode === 'unique') {
-            return t('ux_editor.modal_properties_group_id_not_unique_error');
-          }
-          if (errorCode === 'pattern') {
-            return t('ux_editor.modal_properties_group_id_not_valid');
-          }
-        }}
-        onChange={handleIdChange}
-        renderField={({ fieldProps }) => (
-          <LegacyTextField
-            {...fieldProps}
-            name={`group-id${container.id}`}
-            onChange={(e) => fieldProps.onChange(e.target.value, e)}
-          />
-        )}
-      />
-      <FormField
-        id={container.id}
-        value={container.maxCount > 1}
-        onChange={handleChangeRepeatingGroup}
-        renderField={({ fieldProps }) => (
-          <Switch
-            {...fieldProps}
-            checked={fieldProps.value}
-            size='small'
-            onChange={(e) => fieldProps.onChange(e.target.checked, e)}
-          >
-            {t('ux_editor.modal_properties_group_repeating')}
-          </Switch>
-        )}
-      />
-      {container.maxCount > 1 && (
-        <>
-          <EditGroupDataModelBindings
-            dataModelBindings={container.dataModelBindings}
-            onDataModelChange={handleDataModelGroupChange}
-          />
+  return (
+    <div className={classes.root}>
+      {container.type === ComponentType.RepeatingGroup && (
+        <LegacyFieldSet className={classes.fieldset}>
```

Comment:
Didn't catch this the first time looking through, but this whole section could probably be moved into a separate component. Similar to the custom config setup we have for some of the other components, this could be the custom setup for repeatingGroup.

If you created a new component in
`ux-editor/src/components/config/ComponentSpecificContent/RepeatingGroup`,
and then updated `ComponentSpecificConfig.tsx` with the `RepeatingGroup` component and the custom config component, I think you could delete the `EditFormContainer` entirely and skip the `isContainer` check in `Properties.tsx` 🤔 

What do you think?

Reply:
Seems reasonable 🤩 Fixing now!

Reply:
However, it seems like the `ComponentSpecificConfig` is only rendered when the beta-switch is not enabled? 🤔 

```
{!showComponentConfigBeta && (
        <>
          {renderFromComponentSpecificDefinition(getConfigDefinitionForComponent())}
          <ComponentSpecificContent
            component={component}
            handleComponentChange={handleComponentUpdate}
            layoutName={selectedLayout}
          />
        </>
      )}
```